contracts/governance/StkTruToken.sol

```diff
@@ -401,15 +397,46 @@ contract StkTruToken is VoteToken, StkClaimableContract, ReentrancyGuard {
      * @param token Token to claim rewards for
      */
     function _claim(IERC20 token) internal {
-        farmRewards[token].totalClaimedRewards = farmRewards[token].totalClaimedRewards.add(
-            farmRewards[token].claimableReward[msg.sender]
-        );
-        uint256 rewardToClaim = farmRewards[token].claimableReward[msg.sender];
-        farmRewards[token].claimableReward[msg.sender] = 0;
+        uint256 rewardToClaim = _claim_to_uint256(token);
         require(token.transfer(msg.sender, rewardToClaim));
         emit Claim(msg.sender, token, rewardToClaim);
     }
 
+    /**
+     * @dev Internal claim function that returns the transfer value
+     * Claim rewards for a specific ERC20 token to return in a uint256
+     * @param token Token to claim rewards for
+     */
+    function _claim_to_uint256(IERC20 token) internal returns (uint256) {
```

Comment:
I'm not a fan of having types in function name. We already have it in the function signature. @hal909 @barrutko what do you think?

Reply:
Thanks for the feedback!

Yeah, I was also uncomfortable with the function name for the same reason and spent some time trying to come up with a better one (e.g., _claim_to_tru, _claim_to_token, _partial_claim, _incomplete_claim, _claim_without_transfer). Same with _stake_from_uint256.

I think what we want to ultimately express is that this internal method does a partial update of the contract state but isn't done yet. The change still needs to be carried forward, either by issuing a transaction out or by updating another part of internal state. Unfortunately at its most general everything in the EVM appears to be a uint256, so I agree the type name doesn't say much.

I couldn't come up with anything more concise and legible, so I just sent this PR out to see what the rest of the team thinks.

What do you think of my second-best candidate: _claim_without_transfer() and _stake_without_transfer()? I generally consider it poor style to describe what a function *doesn't* do, but I'm starting to think in this case it might be the most legible description.

Reply:
_claimWithoutTransfer() looks fine, but I'd say it is better to rename current claim to something like `_claimAndTransfer()` and simply call new one `_claim()`. Similarly, `_stake()` seems like an ok name for internal helper for stake method. 

And one more thing, according to style guide https://docs.soliditylang.org/en/latest/style-guide.html function names should be in mixedCase, and not underscored_case.

Generally, the PR looks great!

Reply:
Oh, and when you'll be pushing changes, sync the branch with the current dev, otherwise integration tests with 1Inch will fail :)

Reply:
Thanks!

I think I'll go with _claimWithoutTransfer() for now. If we want claim() to keep a consistent meaning across all smart contracts, then I suspect it would be confusing to have it only zero out claimableReward for this specific contract, but elsewhere mean both zeroing out and transferring. Maybe the overall claim() naming can be revisited at a later date, but I suspect it isn't a priority.

Thanks for the style catch! I should have noticed that myself.

Yep, I'll sync again before pushing.